swift/Samples/TicTacToe/Sources/Main/MainWorkflow.swift

```diff
@@ -76,7 +77,7 @@ extension MainWorkflow {
 
 extension MainWorkflow {
 
-    typealias Rendering = BackStackScreen
+    typealias Rendering = AnyScreen
```

Comment:
I think this could be come a modal container screen in both cases and in the authenticating case, we'd have no modals.

We also could add some "alerts" in that flow if for example we wanted to have an error dialog.

Reply:
I love the "alerts" idea. I was thinking about adding some as well after landing this PR. 

My earlier implementation had the rendering set as ModalContainerScreen but that was the right approach. We usually present modal screen as a form of a subflow in an app and most of the time apps usually have a regular UINavigationController. I wasn't sure if we should wrap all the flow into a ModalContainerScreen just because somewhere down the flow we may end up using a modal. 

I had the same discussion with @dhavalshreyas  about this as well and we settled on this for now but i'm happy to discuss this further and get more opinion on it.

Reply:
We can do it later!

Reply:
Had a discussion regarding this in [slack](https://square.slack.com/archives/CBZJ5V163/p1587151894226900). I will be changing the render type to ModalContainerScreen and will add an empty array for modals for the authenticationWorkflow in order to achieve consistency in screen type. Also, this will help avoid loosing type information by using AnyScreen. Thank you @JustinDSN for the detailed explanation. 🙏